spec/units/peep_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "peep"
+
+describe Peep do
+  describe ".all_peeps" do
+    it "returns an array of all the peeps" do
```

Comment:
Here you're stating that the method should return an array of peeps but then you only actually test for the message of the first peep.

How could you improve this test?